presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoUpgrader.java

```diff
@@ -79,6 +106,37 @@ private static void upgradeUsingRpm(String pathToRpm, boolean checkDependencies)
         if (upgradeRpm != 0) {
             throw new PrestoManagerException("Failed to upgrade presto", upgradeRpm);
         }
-        LOGGER.debug("Successfully upgraded presto");
+    }
+
+    private static File storeConfigFiles()
+            throws PrestoManagerException
+    {
+        File tempFile;
+        try {
+            tempFile = File.createTempFile("PrestoConfigs", ".tar.gz");
+        }
+        catch (IOException e) {
+            throw new PrestoManagerException("Failed to create temp file.", e);
+        }
+        int tarConfigFiles = executeCommand("tar", "-czv", "-f", tempFile.getAbsolutePath(), "-C", CONFIG_DIR, ".");
```

Comment:
Maybe call this `tarResult`? (Same comment for other `executeCommand` return values.